build-scripts/clang-tidy-all.sh

```diff
@@ -24,5 +24,6 @@ do
         -I $BUILD_DIR \
         -I $SUBPROJECTS_DIR/hashmap.c \
         -I $SUBPROJECTS_DIR/inih \
-        -include config.h
+        -include config.h \
+        -Wno-unknown-warning-option
```

Comment:
Since `-I` is including directories and `-include` a file I think they semantically similar and should be next to each other. But I am fine with changing it.

Reply:
Hmm, that's something I didn't realize, sorry.  In that case you are right, have together `-I` and `-included` makes sense. So up to you if you want to do another change or merge as is